src/encoded/static/components/browse/components/FileEntryBlock.js

```diff
@@ -368,13 +369,14 @@ export class FilePairBlock extends React.PureComponent {
     }
 
     render(){
-        const { files, columnHeaders, colWidthStyles, isSingleItem, excludeChildrenCheckboxes, hideNameOnHover, stackDepth } = this.props;
+        const { files, columnHeaders, colWidthStyles, isSingleItem, excludeChildrenCheckboxes, hideNameOnHover, stackDepth, columnClass } = this.props;
         const isReallySingleItem = this.isSingleItem(isSingleItem, files);
         const cls = (
             "s-block file-group keep-label-on-name-hover" +
             (hideNameOnHover ? ' hide-name-on-block-hover' : '') +
             " stack-depth-" + stackDepth
         );
+        const useStyle = colWidthStyles["list:file-group"]; // columnClass here is of parent StackedBlock, not of its children.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['colWidthStyles.list:file-group' is missing in props validation](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4901542)